guide/glossary/address.md

```diff
@@ -84,8 +84,8 @@ Since "legacy" addresses are still in use, some older applications have yet to u
 <div class="center" markdown="1">
 
 {% include image.html
-   image = "/assets/images/guide/payments/request/single-use-address-type.png"
-   retina = "/assets/images/guide/payments/request/single-use-address-type@2x.png"
+   image = "/assets/images/guide/glossary/address/single-use-address-type.png"
+   retina = "/assets/images/guide/glossary/address/single-use-address-type@2x.png"
```

Comment:
```suggestion
   retina = "/assets/images/guide/payments/request/single-use-payment-request-settings-address@2x.png"
```

It was deleted on purpose but I was suppose to replace it with this updated image - sorry about that!

Reply:
OK. I'll update this PR and remove the image I re-added.